lib/message_system.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class MessageSystem
+
+  def send
+    send_message
+    message_body
+  end
+
+  private
+
+  FROM = ENV['TWILIO_FROM'].freeze
+  TO = ENV['TWILIO_TO'].freeze
```

Comment:
you probably don't want the 'to' telephone number to be a constant, as the customers number will change